src/home-page/NoMatch404.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { useHistory } from 'react-router-dom';
+import Typography from '@material-ui/core/Typography';
+import styled from 'styled-components';
+import { Button } from '@material-ui/core';
+import { MainContainer } from '../layout/components/MainContainer';
+
+const NoMatchTypography = styled('div')`
+  display:flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction:column;
+  margin: auto;
+  min-height: 70vh;
+`;
+
+const HeaderTypography = styled(Typography)`
```

Comment:
Mui `Typography` is a way to keep text (fonts, etc.) consistent across our app. Consequently, we should not override font size for individual `Typography` components, but rather choose font sizes for HTML elements (`h1`, `p` etc.) in our Theme and then apply `Typography` variants. 
So the best option here seems to set the desired font size in Theme, say enlarging `h1` and adding a value for `subtitle1`:
```export const theme = createMuiTheme({
  typography: {
    fontFamily: 'Segoe UI, sans-serif',
    h1: {
      fontSize: 40, // <- change to bigger font
      [breakpoints.down('sm')]: {
        fontSize: 30,
      },
      [breakpoints.down('xs')]: {
        fontSize: 24,
      },
    },
    subtitle1: { 
      fontSize: 20, // add a fitting font size for subtitle
    },
    body1: {
      [breakpoints.down('sm')]: {
        fontSize: 14,
      },
      [breakpoints.down('xs')]: {
        fontSize: 13,
      },
    },
```

Reply:
done